services/interfaces/external_interfaces.go

```diff
@@ -47,7 +47,7 @@ type Communication interface {
 
 type Membership interface {
 	MyMemberId() primitives.MemberId
-	RequestOrderedCommittee(ctx context.Context, blockHeight primitives.BlockHeight, randomSeed uint64) ([]primitives.MemberId, error)
+	RequestOrderedCommittee(ctx context.Context, blockHeight primitives.BlockHeight, randomSeed uint64, prevBlockReferenceTime primitives.TimestampSeconds) ([]primitives.MemberId, error)
```

Comment:
Why do we pass both blockHeight and prevBlockReferenceTime? Isn't it redundant?

Reply:
prev block reference time is the time (from management - ethereum) that decides the committee ... it has to come from the previous block so that it is under consensus. This is a requirement.